src/artifacts/OrbitName.tsx

```diff
@@ -24,8 +25,18 @@ export function OrbitName(props: JSX.IntrinsicElements['group']) {
   const { nodes, materials } = useGLTF(
     '/assets/models/cruise-arrival-scene-name-transformed.glb'
   ) as GLTFResult;
+
+  const minScaleFactor = filledVector(2);
+  const maxScaleFactor = filledVector(2.3);
+  const isMobile = window.innerWidth < 768;
```

Comment:
Can we just apply the rescaling at the scene manager level rather than every component?